extension/src/project/readOnlyDocument.ts

```diff
@@ -71,13 +73,21 @@ export class ReadonlyDocument implements vscode.TextDocument {
         return ret;
     }
 
+    //Note to reviewers, this method is used exactly one place. We could update that call to provide 'autolispprj' to the more ambiguous createMemoryDocument()
     static createProject(prjContent: string): ReadonlyDocument {
         let ret = new ReadonlyDocument('');
 
         ret.initialize(prjContent, 'autolispprj');
         return ret;
     }
 
+    // used to create micro memory documents from fragments of AutoLisp code. One application for doing this is clearly identifying the scope of variables.
+    static createMemoryDocument(fileContent: string, languageId: string): ReadonlyDocument {        
```

Comment:
@Sen-real Please help to review it

Reply:
it looks good, basically an upgrade of https://github.com/Autodesk-AutoCAD/AutoLispExt/pull/28/files#diff-b33bf13e4dee64627ec3c1c8acfddf97ca2cbf70824a82fae89605816c69e94bL74

Reply:
Thanks, I will track down that one call and update it to provide its selector